src/models/TestStationDAO.ts

```diff
@@ -35,6 +35,34 @@ export class TestStationDAO {
     }
   }
 
+  /**
```

Comment:
Please correct me if I am wrong, and these methods are different but is it not possible to reuse the method on line 136 that already does this?

getTestStaionIdByPNumber

Although granted, there is a typo in the existing method so that should be renamed to getTestStationIdByPNumber

Reply:
I know the existing logic is only returning the ["testStationId"] specifically, but the rest of the method is identical. So you could return the entire object and then add an extra line to extra the testStationId when the PUT logic is using it.

Reply:
Refactored this so that it is shared now